xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -828,8 +828,15 @@ void CRenderManager::FlipPage(volatile std::atomic_bool& bStop, double pts /* =
         presentmethod = PRESENT_METHOD_BOB;
         invert = true;
       }
+      else if (interlacemethod == VS_INTERLACEMETHOD_IMX_ADVMOTION_DOUBLE)
```

Comment:
> that was the idea, to reuse silly AutoInterlaceMethod

don't reuse code that is silly

do something like
bool CBaseRenderer::WantsDoublePass()

that can be overridden by hw renderer